src/instructlab/config.py

```diff
@@ -19,8 +19,8 @@
 
 DEFAULT_API_KEY = "no_api_key"
 DEFAULT_CONFIG = "config.yaml"
-DEFAULT_MODEL = "merlinite-7b-lab-Q4_K_M"
-DEFAULT_MODEL_PATH = f"models/{DEFAULT_MODEL}.gguf"
```

Comment:
Why are we making changes here? Is it required for the PR goal?

Reply:
The default model needs to match the full path. The -m the user gives and/or the default need to be the fully qualified path to the served model, especially in vllm.

Reply:
something like `merlinite-7b-lab` isn't ever going to be one of the models found when querying the server, only the full path will be recognized